Update README.md - include HBO Max 52.15.0.53 (#126)
riverar/wsa-app-compatibility: README.md

| Grand Theft Auto: San Andreas || ✅
| Guardian Tales | 2.23.2 | 🆖 | Requires GMS
| Hay Day | 1.53.46(1700) | 🆖 | Account synchronization requires GMS, but it can be bypassed with Supercell ID

> | HBO Max | 52.15.0.53 | ⚠️ | Failed to play video | Internal player don't show image or sound

Reviewer: ```suggestion
| HBO Max | 52.15.0.53 | ⚠️ | Failed to play video (internal player fails to display image and play sound).
```
